e2e-tests/src/auth/checkout.npg-payment-auth.integration.test.ts

```diff
@@ -36,15 +36,11 @@ describe("Checkout login and payment flow", () => {
     /**
      * Step 1: Login
      */
-    it("Should successfully login in checkout", async () => {
+    it("Should Login Successfully At Checkout", async() => {
```

Comment:
```suggestion
    it("Should correctly perform login", async() => {
```
🚲 
checkout can be omitted here, it's implicit in repo and test collection name

Reply:
Solved in https://github.com/pagopa/pagopa-checkout-tests/pull/112/commits/d7f84879fbdc6ce3dcbddc32d37616da70ea1fa7